draft-packages/button/KaizenDraft/Button/FilterDrawerButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { Badge } from "@kaizen/draft-badge"
+import classnames from "classnames"
+import * as React from "react"
+import GenericButton, {
+  AdditionalContentProps,
+  GenericProps,
+  LabelProps,
+} from "./components/GenericButton"
+const filterIcon = require("@kaizen/component-library/icons/filter.icon.svg")
+  .default
+const styles = require("./FilterDrawerButton.module.scss")
+
+type FilterButtonProps = {
+  labelText: string
+  numFiltersEnabled: number
```

Comment:
Nitpicky, but I can't think of any other components that use shortened prop names... Maybe `numberOfFiltersEnabled` instead? Or even `badgeNumber`?

Reply:
I really prefer `numberOfFiltersEnabled`, and am actually a big advocate for never abbreviating names. I always worry that people will freak out about long names, but I guess I should embrace it more! I prefer it to `badgeNumber` as it more clearly explains what you're meant to use it for.

Reply:
Sounds good to me